dpdk/vr_dpdk_ethdev.c

```diff
@@ -484,3 +510,13 @@ vr_dpdk_ethdev_init(struct vr_dpdk_ethdev *ethdev)
 
     return 0;
 }
+
+/* Release ethernet device */
+int
+vr_dpdk_ethdev_release(struct vr_dpdk_ethdev *ethdev)
+{
+    /* TODO */
```

Comment:
I think a function reversing dpdk_ethdev_queues_setup() is needed here. I will implement it and commit it soon.